crates/e2e/src/nodes/forked_node.rs

```diff
@@ -61,4 +61,16 @@ impl<T: Transport> ForkedNodeApi<T> {
                 .execute("anvil_setBalance", vec![json_address, json_balance]),
         )
     }
+
+    // Makes GPv2 auth contract always return `true` to make any solver to be
+    // whitelisted.
+    pub fn set_mocked_settle(&self) -> CallFuture<(), T::Out> {
+        let gpv2_auth_address_json =
+            serde_json::json!("0x2c4c28DDBdAc9C5E7055b4C863b72eA0149D8aFE");
+        let bytecode_json = serde_json::json!("0x600160005260206000F3");
+        CallFuture::new(
+            self.transport
+                .execute("anvil_setCode", vec![gpv2_auth_address_json, bytecode_json]),
+        )
+    }
```

Comment:
Why is this necessary now? Can't we just use an existing solver address or whitelist one we need? How is the [existing e2e test](https://github.com/cowprotocol/services/blob/3a6b13e0669e629f9a0f36c5f10bd85f648076a9/crates/e2e/tests/e2e/limit_orders.rs#L491) achieving this?